test/aggregate.test.js

```diff
@@ -12,10 +12,10 @@ describe("Aggregate{}", () => {
   const config = {
     initialState: 0,
     eventHandlers: {
-      counterIncremented: (state, event) => state + 1
+      counterIncremented: (state) => state + 1
     },
     commandHandlers: {
-      incrementCounter: (state, command) => Success([{ type: "counterIncremented" }])
+      incrementCounter: () => Success([{ type: "counterIncremented" }])
     }
```

Comment:
That's a good point. I just went blindly removing unused args. How about ignoring arguments that match [`^_`](http://eslint.org/docs/rules/no-unused-vars#argsignorepattern) (just like rubocop does in when enforcing the ruby style-guide)?